cc3d/core/PySteppables.py

```diff
@@ -1706,29 +1706,35 @@ def get_fpp_anchor_by_cell_and_id(self, cell: CompuCell.CellG, anchor_id: int) -
         """
         return self.fpp_anchor_inventory.getAnchor(cell, anchor_id)
 
-    def get_fpp_links_by_cell(self, _cell: CompuCell.CellG) -> CompuCell.FPPLinkList:
+    def get_fpp_links_by_cell(self, _cell: CompuCell.CellG) -> Optional[FocalPointPlasticityLinkList]:
         """
         Get list of links by cell
         :param _cell: cell
         :return: links
         """
-        return self.fpp_link_inventory.getCellLinkList(_cell)
+        if self.focal_point_plasticity_plugin is None:
+            return None
+        return FocalPointPlasticityLinkList(self.focal_point_plasticity_plugin, _cell)
 
-    def get_fpp_internal_links_by_cell(self, _cell: CompuCell.CellG) -> CompuCell.FPPInternalLinkList:
+    def get_fpp_internal_links_by_cell(self, _cell: CompuCell.CellG) -> Optional[FocalPointPlasticityInternalLinkList]:
         """
         Get list of internal links by cell
         :param _cell: cell
         :return: internal links
         """
-        return self.fpp_internal_link_inventory.getCellLinkList(_cell)
+        if self.focal_point_plasticity_plugin is None:
+            return None
```

Comment:
Since many of those get_fpp_internal_links_xxx function are used in loop context we may consider raising StopIteration exception instead of returning None.